contrib/babelfishpg_common/src/collation.c

```diff
@@ -1180,6 +1180,9 @@ int get_persist_collation_id(Oid coll_oid)
 	bool found_coll;
 	int collidx;
 
+	if (ht_oid2collid == NULL)
```

Comment:
did we make sure that there are no other places which require similar check?

Reply:
Yes, I've verified the same.